lib/charms/prometheus_k8s/v0/prometheus.py

```diff
@@ -462,13 +458,9 @@ def alerts(self):
             if not relation.units:
                 continue
 
-            alert_rules = json.loads(
-                relation.data[relation.app].get("alert_rules", "{}")
-            )
```

Comment:
There is a lot of this which is clearly a `black` thing - can you seperate the work you're doing from the `black` formatting to make the diff a bit more readable?

Reply:
it is already divided into two separate commits

Reply:
let me drop that linting commit for now (gh action may fail)